app/controllers/tags_controller.rb

```diff
@@ -248,9 +248,8 @@ def update
     new_tag_type = params[:tag].delete(:type)
 
     # Limiting the conditions under which you can update the tag type
-    if @tag.can_change_type? && %w(Fandom Character Relationship Freeform UnsortedTag).include?(new_tag_type)
-      @tag = @tag.recategorize(new_tag_type)
-    end
+    types = logged_in_as_admin? ? Tag::TYPES : Tag::USER_DEFINED
```

Comment:
This will also allow rating, warning, and category tags to be renamed by admins. Let's hold off on that.